capstone-ui/src/Components/App/App.jsx

```diff
@@ -71,7 +72,12 @@ export default function App() {
             </div>
             <div className="content">
               <Routes>
-                <Route path='/' element={ user ? (<Feed loggedin={true}/>) : (<Feed loggedin={false}/>)}/>
+                <Route path='/' element={ user ? (<div className='loggedin-home'>
```

Comment:
Can we wrap this in a `Home` component and pass in `<Home loggedIn={user != null}/>`. This way it is more readable and we can separate Home screen features from App. It will also prevent duplicate code here e.g. `<Feed loggedIn={<value>} />` when the user could have been passed instead. For `Recommendations` component, it can be checked if needed in `Home` component.